gatsby-config.js

```diff
@@ -3,16 +3,18 @@ require('dotenv').config();
 module.exports = {
   siteMetadata: {
     title: 'Gatsby Default Starter',
+    languages: ['nl', 'en-US'],
+    defaultLocale: 'nl',
   },
   plugins: [
     'gatsby-plugin-react-helmet',
     'gatsby-plugin-styled-components',
-    'gatsby-plugin-react-helmet',
     {
       resolve: 'gatsby-source-contentful',
       options: {
         spaceId: process.env.CONTENTFUL_SPACE_ID,
         accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
+        environment: 'translations',
```

Comment:
This environment is for getting the translation environment in contentful. When you remove this line, you will get the default contentful data.